src/registrar/templates/domain_users.html

```diff
@@ -57,7 +57,7 @@ <h2>Invitations</h2>
       <tbody>
         {% for invitation in domain.invitations.all %}
         <tr>
-          <th scope="row" role="rowheader" data-sort-value="{{ user.email }}" data-label="Email">
+          <th scope="row" role="rowheader" data-sort-value="{{ invitation.user.email }}" data-label="Email">
```

Comment:
![image](https://github.com/cisagov/getgov/assets/109625347/1b1cc962-ff43-4805-b085-63d3f2653a4f)
is this coloration change expected with using sort? It seems off to me. tagging @gabydisarli or @Katherine-Osos

Reply:
@abroddrick This is consistent with staging as well:
![image](https://github.com/cisagov/getgov/assets/141044360/86ab79de-1e26-48c4-bec9-d55a1c577f58)

I don't think this is related to this ticket as it exists there. That said, I think this may be something to dig deeper into in another UI ticket - there are a few things that may need a second glance in that

Reply:
cc: @Katherine-Osos / @gabydisarli